rocketpy/plots/plot_helpers.py

```diff
@@ -0,0 +1,68 @@
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from matplotlib.figure import Figure
+
+from ..tools import get_matplotlib_supported_file_endings
+
+SAVEFIG_DPI = 300
+
+
+def show_or_save_plot(filename=None):
+    """Shows or saves the current matplotlib plot. If a filename is given, the
+    plot will be saved, otherwise it will be shown.
+
+    Parameters
+    ----------
+    filename : str | None, optional
+        The path the plot should be saved to, by default None. Supported file
+        endings are: eps, jpg, jpeg, pdf, pgf, png, ps, raw, rgba, svg, svgz,
+        tif, tiff and webp (these are the formats supported by matplotlib).
+    """
+    if filename == None:
+        plt.show()
+    else:
+        # Warn if file ending is not supported
+        file_ending = Path(filename).suffix
+        if file_ending not in get_matplotlib_supported_file_endings():
+            warnings.warn(
+                f"Warning: Unsupported file ending '{file_ending}'!", UserWarning
+            )
```

Comment:
Since an unsupported file ending will cause an error, should we not raise a ValueError instead? For some reason, the warnings are only being shown once for me, all calls that follow throw a ValueError by `plt.savefig`.